src/routes/Things.js

```diff
@@ -47,8 +49,54 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Things', (req, res) => {
-  res.status(200).send();
+const resource = 'Things';
+
+router.get('/', (req, res) => {
+  db().then((dbInstance) => {
+    dbInstance.Things.findAll().then(things => {
```

Comment:
Let's do

`models.Things.findAll({ attributes: { exclude: ['id', 'createdAt', 'updatedAt'] }}).then(things => {`

and remove

https://github.com/mozilla-sensorweb/sensorthings/pull/4/files#diff-d0f5704ae0738a7bd1f54aff42ddcb41R34

The DB is probably faster than us doing this kind of filtering. Although it is probably worth to check if the filtering is done as part of the SQL statement or sequelize does it in JS.